flocker/node/agents/test/test_blockdevice.py

```diff
@@ -126,10 +119,21 @@
 from ....common.test.test_thread import NonThreadPool, NonReactor
 from ....common import RACKSPACE_MINIMUM_VOLUME_SIZE
 
-CLEANUP_RETRY_LIMIT = 10
-LOOPBACK_ALLOCATION_UNIT = int(MiB(1).to_Byte().value)
-# Enough space for the ext4 journal:
-LOOPBACK_MINIMUM_ALLOCATABLE_SIZE = int(MiB(16).to_Byte().value)
+from ..testtools import (
+    FakeCloudAPI,
+    detach_destroy_volumes,
+    fakeprofiledloopbackblockdeviceapi_for_test,
+    loopbackblockdeviceapi_for_test,
+    # NB: Think carefully before moving this.
+    # Backend driver docs used to instruct developers to import this from here.
```

Comment:
I created https://clusterhq.atlassian.net/browse/FLOC-4308.

If you don't mind, I'll link to it in my next branch rather than wait for another test run.